nni/retiarii/oneshot/pytorch/base_lightning.py

```diff
@@ -147,34 +153,40 @@ def no_default_hook(module: nn.Module, name: str, memo: dict[str, Any], mutate_k
 class BaseOneShotLightningModule(pl.LightningModule):
 
     _mutation_hooks_note = """mutation_hooks : list[MutationHook]
-        Mutation hooks are callable that inputs an Module and returns a :class:`BaseSuperNetModule`.
+        Extra mutation hooks to support customized mutation on primitives other than built-ins.
+
+        Mutation hooks are callable that inputs an Module and returns a
+        :class:`BaseSuperNetModule <nni.retiarii.oneshot.pytorch.supermodule.base.BaseSuperNetModule>`.
```

Comment:
long doc to explain mutation hooks without any simple example. a simple example could be much clearer.

Reply:
I can't think any "simple" example. Maybe I can add some reference to existing hooks.